leapp/cli/report/__init__.py

```diff
@@ -0,0 +1,79 @@
+"""
+This module is currently single workflow specific (upgrades). After introducing other use cases we can make it
+more robust
+"""
+
+import os
+
+from leapp.exceptions import CommandError
+from leapp.utils.clicmd import command, command_opt, command_arg
+from leapp.utils.report import fetch_upgrade_report_messages
+from leapp.cli.upgrade import fetch_last_upgrade_context
+
+from jinja2 import Template
+import json
+
+
+@command('report', help='Create report for last upgrade execution (default) or specific execution id using --id ID')
+@command_opt('id')
+@command_opt('--format')
+def report(args):
+    if os.getuid():
+        raise CommandError('This command has to be run under the root user.')
+
+    id = args.id or fetch_last_upgrade_context()[0]
+
+    if not id:
+        raise CommandError('No previous Leapp upgrade run of the requested context {} found'.format(id))
+
+    messages = fetch_upgrade_report_messages(id)
+    if not messages:
+        raise CommandError('No upgrade report messages found for context {}'.format(id))
+
+    # FIXME: add proper default and choices parametrs for the format option
+    if args.format not in ('html', 'plaintext'):
+        print('No / not supported renderer provided as --format (available renderers: "html", "plaintext"). '
```

Comment:
Please use logging instead of print

Reply:
@fernflower For starting to use logging, one would have to setup logging first. This command hasn't it set up just yet. So probably not too bad.